src/content/basics/giant-swarm-support/index.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Giant Swarm Support"
+description = "An explanation of how our support service works."
+date = "2019-04-16"
+type = "page"
+weight = 20
+categories = ["basics"]
++++
+
+# Giant Swarm Support
+
+Our support service is one of the key differentiators from our competitors. We have built a platform which lets our customer spin up as many clusters as they need. These clusters are then always in the same predictable state. This becomes especially valuable once you need to run multiple clusters for different teams, environments, configurations or in multiple locations.
+
+We are always at the side of our customers during their whole journey. For us, having a great platform is as important as having someone to contact when problems occur. The following is a brief description of our support process.
+
+## Customer Support
+
+Our customer support relies on close interactions via Slack, to get bi-directional feedback as quickly as possible. This is our first line of contact to customers, who can ask all kinds of questions, not only in relations to the platform itself, but many more around their whole Cloud Native journey.
+
+Our support shifts go from 08:00 to 18:00 on weekdays, but we also often answer questions well outside those times. We have a rotating shift for support across the team, which is why we keep support focussed on those channels with clear internal handovers.
```

Comment:
`focused` but I had to check that one! 😄

Reply:
my grammar corrector says both are ok 🤷‍♀️ maybe English variants?

Reply:
`focused` is 🇺🇸 english which is what we use in sig-content.

Reply:
ah ok good to know :D